hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -1588,6 +1588,16 @@
 	@SuppressWarnings("JavaDoc")
 	String HBM2DDL_SCRIPTS_CREATE_TARGET = "javax.persistence.schema-generation.scripts.create-target";
 
+	/**
+	 * For cases where the {@value #HBM2DDL_SCRIPTS_ACTION} value indicates that schema commands should
+	 * be written to DDL script file, specifies specifies if schema commands should be appended to the end of the file rather than written at the beginning of the file.
+	 *
+	 * Values are: {@code true} for appending schema commands to the end of the file, {@code false} for schema writing commands at the beginning.
```

Comment:
'for schema writing commands' -> 'for writing schema commands'?

Reply:
Thanks @NathanQingyangXu , fixed it!